src/NotepadNext/docks/SearchResultsDock.cpp

```diff
@@ -210,3 +211,18 @@ void SearchResultsDock::updateSearchStatus()
     if (currentFile)
         currentFile->setText(0, QStringLiteral("%1 (%L2 hits)").arg(currentFilePath).arg(totalFileHitCount));
 }
+
+void SearchResultsDock::on_pb_copyResults_released()
+{
+    QClipboard *clipboard = QGuiApplication::clipboard();
+    QString string;
+
+    QTreeWidgetItemIterator it(ui->treeWidget);
+    while (*it) {
+        string.append((*it)->text(0) + ' ' + (*it)->text(1) + '\n');
```

Comment:
Should `\r\n` be used on Windows? Not sure if it makes a difference. Maybe the clipboard normalizes line ending.

Reply:
I have been unable at this point to locate any information on line ends for the clipboard, and I don't have access to windows to test it, but it is a good question :)

Reply:
I'll try it out on windows and see what happens :)